[2025-06-28T10:22:20.905+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator.external_python manual__2025-06-28T10:22:07.401451+00:00 [queued]>
[2025-06-28T10:22:20.924+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator.external_python manual__2025-06-28T10:22:07.401451+00:00 [queued]>
[2025-06-28T10:22:20.926+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-06-28T10:22:20.996+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonExternalDecoratedOperator): external_python> on 2025-06-28 10:22:07.401451+00:00
[2025-06-28T10:22:21.031+0000] {standard_task_runner.py:57} INFO - Started process 123 to run task
[2025-06-28T10:22:21.041+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator', 'external_python', 'manual__2025-06-28T10:22:07.401451+00:00', '--job-id', '11', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_python_operator.py', '--cfg-path', '/tmp/tmpncfdkiwc']
[2025-06-28T10:22:21.048+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask external_python
[2025-06-28T10:22:21.234+0000] {task_command.py:416} INFO - Running <TaskInstance: example_python_operator.external_python manual__2025-06-28T10:22:07.401451+00:00 [running]> on host 4eaa1e3302ef
[2025-06-28T10:22:21.419+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator' AIRFLOW_CTX_TASK_ID='external_python' AIRFLOW_CTX_EXECUTION_DATE='2025-06-28T10:22:07.401451+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-28T10:22:07.401451+00:00'
[2025-06-28T10:22:25.427+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python /tmp/tmdyyz7glfy/script.py /tmp/tmdyyz7glfy/script.in /tmp/tmdyyz7glfy/script.out /tmp/tmdyyz7glfy/string_args.txt /tmp/tmdyyz7glfy/termination.log
[2025-06-28T10:22:25.443+0000] {process_utils.py:186} INFO - Output:
[2025-06-28T10:22:30.807+0000] {process_utils.py:190} INFO - Running task via /usr/local/bin/python
[2025-06-28T10:22:30.826+0000] {process_utils.py:190} INFO - Sleeping
[2025-06-28T10:22:30.834+0000] {process_utils.py:190} INFO - Please wait...
[2025-06-28T10:22:31.957+0000] {process_utils.py:190} INFO - Please wait...
[2025-06-28T10:22:32.919+0000] {process_utils.py:190} INFO - Please wait...
[2025-06-28T10:22:33.918+0000] {process_utils.py:190} INFO - Please wait...
[2025-06-28T10:22:34.936+0000] {process_utils.py:190} INFO - Finished
[2025-06-28T10:22:35.618+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-28T10:22:35.684+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=external_python, execution_date=20250628T102207, start_date=20250628T102220, end_date=20250628T102235
[2025-06-28T10:22:35.764+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-28T10:22:35.824+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
